DataFormats/HGCalReco/interface/HGCalSoARecHits.h

```diff
@@ -14,13 +14,17 @@ GENERATE_SOA_LAYOUT(HGCalSoARecHitsLayout,
                     SOA_COLUMN(float, dim2),
                     SOA_COLUMN(float, dim3),
                     SOA_COLUMN(int, layer),
-                    SOA_COLUMN(float, weight),
+                    SOA_COLUMN(float, energy),
+                    SOA_COLUMN(float, mipEnergy),
                     SOA_COLUMN(float, sigmaNoise),
                     SOA_COLUMN(unsigned int, recHitIndex),
+                    SOA_COLUMN(uint16_t, flags),
                     SOA_COLUMN(uint32_t, detid),
                     SOA_COLUMN(float, time),
                     SOA_COLUMN(float, timeError))
 
 using HGCalSoARecHits = HGCalSoARecHitsLayout<>;
 
+enum HGCalRecHitFlags { Normal = 0x0, EnergyInvalid = 0x1, TimeInvalid = 0x2 };
```

Comment:
just a small suggestion:
```suggestion
enum HGCalRecHitFlags { kNormal = 0x0, kEnergyInvalid = 0x1, kTimeInvalid = 0x2 };
```

Reply:
done